xbmc/cores/VideoPlayer/Interface/StreamInfo.h

```diff
@@ -30,6 +30,16 @@ enum StreamFlags
   FLAG_STILL_IMAGES = 0x100000
 };
 
+enum StreamHdrType
```

Comment:
c++ version is: `enum class StreamHdrType`. Type below should probably be `StreamHdrType m_iHdr_Type` instead of `int m_iHdr_Type`

Reply:
Thanks for the heads up.  Changed.